joomla/joomla-cms
# [4.0] ES6 please, com_templates

Pull Request for Issue # .

### Summary of Changes
Refactor the com_templates script to ES6


### Testing Instructions
Check that the functionality on the template manager (overrides, new folder, edit file etc) is still intact 


### Expected result

<img width="1644" alt="screenshot 2018-08-07 at 00 54 18" src="https://user-images.githubusercontent.com/3889375/43744983-d007b4f4-99dc-11e8-9c30-5a3c743d746c.png">


### Actual result



### Documentation Changes Required
